### DISCUSSION TITLE
Copilot free plan stopped working and regain access

### DISCUSSION THREAD

**Discussion by SakibSumon** [community/community#45511] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I had copilot free plan for students. but on January 9, 2023 it suddenly stopped working. and that day I bought copilot. but I heard that students whose copilot stopped working are now enabled. My question is, am I still able to get the free plan if I cancel the paid version?

**Comment by jge162** [comment, marked as answer]
hello @SakibSumon [copilot support](https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot) you can go to the bottom right of page and click contact us. support should be able to answer your question or, the answer is in support page.

**Reply by SakibSumon** [reply]
hello @jge162, I contacted them through support and they said I'm still eligible for the free plan of copilot. Thank you.

**Reply by jge162** [reply]
glad to have helped, you can close post if you wish as answered. Best to you, have a great week.
